src/Utils/MX/Methods/LoadRandomMap.as

```diff
@@ -114,6 +114,19 @@ namespace MX
             return;
         }
 
+        if (
+            (((RMC::IsRunning || RMC::IsStarting) && PluginSettings::CustomRules && PluginSettings::MapAuthorNameNeedsExactMatch) 
+            || (!RMC::IsRunning && !RMC::IsStarting && PluginSettings::MapAuthorNameNeedsExactMatch)) && PluginSettings::MapAuthor != ""
+        ) {
+            string author = map.Username.ToLower();
+            print(PluginSettings::MapAuthorNamesArr.Find(author));
+            if (PluginSettings::MapAuthorNamesArr.Find(author) == -1) {
+                Log::Warn("Map author does not match, retrying...");
+                PreloadRandomMap();
+                return;
+            }
```

Comment:
Something's not working, tried searching with my username in lowercase (my MX username is first letter capital case) and returns "does not match"

![image](https://github.com/GreepTheSheep/openplanet-MXRandom/assets/42576124/e529b2e7-376a-476f-a9c3-d89871daf29e)

Reply:
should be fixed now